packages/recomponents/src/components/r-button/r-button.scss

```diff
@@ -2,7 +2,6 @@
     display: inline-block;
     height: auto;
     box-sizing: border-box;
-    text-transform: capitalize;
```

Comment:
I'm not sure about this but, did you try [first-letter](url)? Maybe that could be a much simpler solution.

Reply:
Great, I'd like to implement it in another PR

Reply:
Why another PR instead update of this one?

Reply:
that's totally a different way

Reply:
We can compare which one is better then choose it

Reply:
Will this code below (if added to `r-button.scss`) give you same result as your original JS way?

```css
&:first-letter {
    text-transform: capitalize;
}
```

Reply:
yeah, it works for r-button, but not for r-icon-button

Reply:
I guess is that the first letter in r-icon-button, it could be an icon.

Reply:
Yes, definitely. The only workaround I found so far is to wrap all content of slot into `<span/>` + apply flex to convert each slot into row (this this apply first-letter rule)